### PR TITLE
Scrolling performance improvements

### PR DESCRIPTION
**Pull request by wire-android**
This relies on SE changes in https://github.com/wireapp/wire-android-sync-engine/pull/83

Please see comments in individual commits.
#### APK
[Download build #8634](http://192.168.10.18:8080/job/Pull%20Request%20Builder/8634/artifact/build/artifact/wire-dev-PR705-8634.apk)
[Download build #8644](http://192.168.10.18:8080/job/Pull%20Request%20Builder/8644/artifact/build/artifact/wire-dev-PR705-8644.apk)

### CODE REVIEW

**Comment by AndroidBob** [comment]
Can one of the admins verify this patch?

**Comment by deanrobertcook** [comment]
build again

File: app/src/main/scala/com/waz/zclient/messages/MessagesListView.scala [hunk 1]
```diff
@@ -56,15 +56,16 @@ class MessagesListView(context: Context, attrs: AttributeSet, style: Int) extend
...
+  messageActionsController.messageToReveal {
+    case Some(messageData) =>
+      adapter.positionForMessage(messageData) .foreach { pos =>
+        if (pos >= 0) {
+          scrollController.targetPosition = Some(pos)
+          scrollController.scrollToPositionRequested ! pos
+          messageActionsController.messageToReveal ! None
+        }
+      } (Threading.Ui)
+    case None => Signal.empty[Int]
```
Comment (deanrobertcook) [inline app/src/main/scala/com/waz/zclient/messages/MessagesListView.scala]: I guess we don't need to return anything here, do we?

**Comment by deanrobertcook** [comment]
build again

**Comment by JPonte** [comment]
build again

**Pull request merged**